File: java/io/NoLoad.java
package io;

public final class NoLoad {
    private static final int NUMBER = 500000;

    public static void run(int times) {
        var x = 0;

        for(var i = 0; i <= times; i++) {
            x = i * i;
        }

        System.out.println(x);
    }

    public static void main(String[] args) {
        run(NUMBER);
    }
}

File: javascript/io/NoLoad.js
'use strict';

const NUMBER = 500000;

function run(times) {
    var x = 0;

    for (var i = 0; i <= times; i++) {
        x = i * i;
    }

    console.log(x);
}

(async function() {
    run(NUMBER);
})();
